pkg/moov/a_test.go

```diff
@@ -97,6 +99,13 @@ func NoResponseError(t *testing.T, err error) {
 	require.NoError(t, err)
 }
 
+func PrettyDebug(t *testing.T, a any) {
+	b, err := json.MarshalIndent(a, "  ", "  ")
+	require.NoError(t, err)
+
+	t.Logf("\n%s\n", string(b))
+}
```

Comment:
Needs removed.

Reply:
Thats to log the message in the test failures, its not a `fmt.Printf` its a log error in a test file.